help/destinations/catalog/email-marketing/salesforce-marketing-cloud-account-engagement.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Salesforce Marketing Cloud Account Engagement
+description: The Salesforce Marketing Cloud Account Engagement (formerly known as Pardot) destination allows you to export your account data and activate it within Salesforce Marketing Cloud Account Engagement for your business needs.
+---
+# [!DNL Salesforce Marketing Cloud Account Engagement] connection
+
+[[!DNL Salesforce Marketing Cloud Account Engagement]](https://www.salesforce.com/products/marketing-cloud/marketing-automation/) *(formerly known as [!DNL Pardot])* allows you to capture, track, score and grade leads. You can also design lead tracks for all stages of the pipeline for targeted market segments and customer groups through email drip campaigns and lead management with nurturing, scoring and campaign segmentation.
+
+Compared to [!DNL Salesforce Marketing Cloud Engagement] which is more oriented towards **B2C** marketing, [!DNL Marketing Cloud Account Engagement] is ideal for **B2B** use cases involving multiple departments and decision makers which require longer sales and decision cycles. In addition, you also maintain closer proximity and integration with your CRM to make appropriate sales and marketing decisions. *Note, Experience Platform also has connections for [!DNL Salesforce Marketing Cloud Engagement], you can check them on the [[!DNL Salesforce Marketing Cloud]](/help/destinations/catalog/email-marketing/salesforce-marketing-cloud.md) and [[!DNL (API) Salesforce Marketing Cloud]](/help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md) pages.*
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL Salesforce Account Engagement API > Prospect Upsert by Email]](https://developer.salesforce.com/docs/marketing/pardot/guide/prospect-v5.html#prospect-upsert-by-email) endpoint, to **add or update your leads** after activating them within a new [!DNL Marketing Cloud Account Engagement] segment.
+
+[!DNL Marketing Cloud Account Engagement] uses [OAuth 2 with Authorization Code](/help/destinations/destination-sdk/functionality/authentication/oauth2-authentication.md#authorization-code) to communicate with the [!DNL Account Engagement] API. Instructions to authenticate to your [!DNL Marketing Cloud Account Engagement] instance are further below, in the [Authenticate to destination](#authenticate) section.
```

Comment:
I missed this part in my reviews, but the link in this part is not truly relevant to customers: 

[!DNL Marketing Cloud Account Engagement] uses [OAuth 2 with Authorization Code](/help/destinations/destination-sdk/functionality/authentication/oauth2-authentication.md#authorization-code) to communicate with the [!DNL Account Engagement] API.

This is more of an internal implementation detail. I'll fix this before I publish the page, but just something to keep in mind for future docs.